src/deluge/gui/views/view.cpp

```diff
@@ -158,7 +158,7 @@ ActionResult View::buttonAction(deluge::hid::Button b, bool on, bool inCardRouti
 				endMidiLearnPressSession();
 			}
 		}
-		else if (currentUIMode == UI_MODE_NONE) {
+		else if (currentUIMode == UI_MODE_NONE || currentUIMode == UI_MODE_RECORD_COUNT_IN) {
```

Comment:
does tap tempo work during the count in? I feel like changing tempo mid count in is unlikely to be intentional

Reply:
> does tap tempo work during the count in? I feel like changing tempo mid count in is unlikely to be intentional

Good catch, just blocked tap tempo during record count in